go.mod

```diff
@@ -6,7 +6,7 @@ toolchain go1.21.0
 
 require (
 	github.com/celestiaorg/nmt v0.19.0
-	github.com/celestiaorg/quantum-gravity-bridge v1.3.0
+	github.com/celestiaorg/quantum-gravity-bridge v2.0.0+incompatible
```

Comment:
I think the `+incompatible` suffix is because we cut [v2.0.0](https://github.com/celestiaorg/quantum-gravity-bridge/releases/tag/v2.0.0) of QGB without following the convention of adding the `v2` suffix in the qgb module path.

Reply:
Yes, also because there is no `go.mod` in the QGB repo

Reply:
How come there's no go.mod?

Reply:
https://github.com/celestiaorg/celestia-app/pull/2460#issuecomment-1717843258

Reply:
Created https://github.com/celestiaorg/quantum-gravity-bridge/issues/198